spark-indexer/src/main/java/org/uniprot/store/spark/indexer/proteome/converter/DatasetProteomeEntryConverter.java

```diff
@@ -19,26 +30,338 @@
+    public static final String GENOME_ANNOTATION_SOURCE = "genomeAnnotationSource";
+    public static final String GENOME_ANNOTATION_URL = "genomeAnnotationUrl";
+    public static final String GENOME_ASSEMBLY = "genomeAssembly";
+    public static final String GENOME_ASSEMBLY_SOURCE = "genomeAssemblySource";
+    public static final String GENOME_ASSEMBLY_URL = "genomeAssemblyUrl";
+    public static final String GENOME_REPRESENTATION = "genomeRepresentation";
+    public static final String COMPONENT = "component";
+    public static final String NAME = "_name";
+    public static final String PROTEIN_COUNT = "_proteinCount";
+    public static final String DESCRIPTION = "description";
+    public static final String ANNOTATION_SCORE = "annotationScore";
+    public static final String NORMALIZED_ANNOTATION_SCORE = "_normalizedAnnotationScore";
+    public static final String REFERENCE = "reference";
+    public static final String CITATION = "citation";
+    public static final String TYPE = "_type";
+    public static final String DATE = "_date";
+    public static final String TITLE = "title";
+    public static final String AUTHOR_LIST = "authorList";
+    public static final String PERSON = "person";
+    public static final String DB_REFERENCE = "dbReference";
+    public static final String FIRST = "_first";
+    public static final String LAST = "_last";
+    public static final String VOLUME = "_volume";
+    public static final String DB = "_db";
+    public static final String SCORES = "scores";
+    public static final String VALUE = "_VALUE";
+    public static final String BIO_SAMPLE_ID = "biosampleId";
+    public static final String GENOME_ACCESSION = "genomeAccession";
+    public static final String PROPERTY = "property";
+    public static final String VALUE_LOWER = "value";
+    public static final String CONSORTIUM = "consortium";
+    public static final String ID = "_id";
+    public static final String ISOLATE = "isolate";
+    public static final String REDUNDANT_TO = "redundantTo";
+    public static final String PANPROTEOME = "panproteome";
+    public static final String REDUNDANT_PROTEOME = "redundantProteome";
+    public static final String EXCLUDED = "excluded";
+    public static final String SIMILARITY = "similarity";
+    public static final String EXCLUSION_REASON = "exclusionReason";
 
     @Override
     public ProteomeEntry call(Row row) throws Exception {
         ProteomeEntryBuilder builder = new ProteomeEntryBuilder();
-        builder.proteomeId(row.getString(row.fieldIndex("upid")));
-        // put upid and taxonomy from xml
+        // upid
+        builder.proteomeId(row.getString(row.fieldIndex(UPID)));
+        // taxonomy
         Taxonomy taxonomy =
-                new TaxonomyBuilder().taxonId(row.getInt(row.fieldIndex("taxonomy"))).build();
+                new TaxonomyBuilder().taxonId(row.getInt(row.fieldIndex(TAXONOMY))).build();
         builder.taxonomy(taxonomy);
```

Comment:
This is not to be fixed in this JIra. but it is just a reminder that at some point in the flow. we need to join with Taxonomy data and add values for scientificName, commonName and mnemonic. See the example of the current final ProteomeEntry returned in the rest: https://rest.uniprot.org/proteomes/UP000005640.json